backend/no_waste.py

```diff
@@ -0,0 +1,75 @@
+import os # biblioteka systemowa
+from flask import Flask, json, abort
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
+# glowny modul flask 
+
+#konfiguracja bazy danych
+basedir = os.path.abspath(os.path.dirname(__file__)) # katalog aplikacji
+# __file__ <- sciezka biezacego pliku
+# os.path.dirname <- wez katalog ze sciezki (np /a/b/c -> /a/b)
+# os.path.abspath <- rozwin sciezke np ~/a/b/c -> /home/user/a/b/c
+
+app.config['SECRET_KEY'] = 'hard to guess string'
+app.config['SQLALCHEMY_DATABASE_URI'] =\
+    'sqlite:///' + os.path.join(basedir, 'data.sqlite')
```

Comment:
I know this is only an initial commit, but didn't we talk about using Mongo? Asking because AFAIR you cannot use SQLAlchemy with Mongo.

Reply:
I thought that we will replace SQLAlchemy with Mongo later? We can connect backend and frontend and later change to mongo? Cause frontend will not see the difference anyway?

Reply:
As long as we keep the same REST API there is not different to frontend. I was just curious 😉